### PR TITLE
Terraform

### PR DESCRIPTION
**Pull request by karineiji**

### CODE REVIEW

**Comment by karineiji** [comment]
cant get access to push from terminal, so uploaded by hands

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    values = ["ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
```
Comment (chapayevdauren) [inline main.tf:19]: @karineiji can you explain why we need this block?

**Comment by chapayevdauren** [comment]
@karineiji please upload terraform.tfstate file and screenshots with resources created in AWS

File: main.tf (hunk 1 above)
Comment (karineiji) [reply main.tf:19]: they are parameters of our EC2 hardware. it is our image ubuntu-focal-20.04-amd64-server and we use hard virtual machine.

**Comment by karineiji** [comment]
I don't have tfstate file. I suppose because it is remote

**Pull request closed without merging**